### PR TITLE
Add the proverb exercise.

### PR DESCRIPTION
**Pull request by MichaelBunker**
See the specification: https://github.com/exercism/problem-specifications/tree/main/exercises/proverb

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -139,6 +139,17 @@
...
+      {
+        "slug": "proverb",
+        "name": "Proverb",
+        "uuid": "75519a64-8237-49e5-927f-c5974816258e",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2,
+        "topics": [
+          "strings"
+        ]
```
Comment (neenjaw) [inline config.json:151]: As mentioned in https://github.com/exercism/php/pull/534, this property is deprecated

**Review by neenjaw — APPROVED** [review]
Looks good, once you look at the feedback for https://github.com/exercism/php/pull/534 I think a few of the same comments can be applied here, but I don't want to nag.

Happy to merge this optimistically, let me know what you'd like to do

File: config.json (hunk 1 above)
Comment (MichaelBunker) [reply config.json]: Topics has been removed.

**Comment by MichaelBunker** [comment]
@neenjaw I removed topics and the static implementation per your feedback. This is ready for another review.

**Review by neenjaw — APPROVED** [review]

**Pull request merged**